blog/2023/06/PSA-dont-use-threadid.md

```diff
@@ -212,6 +212,17 @@ MultiThreadedCaches.jl on the other hand attempts to make the `states[threadid()
 
 \\
 
+[^yielding]: ~~~<h4>~~~Don't try to reason about yielding~~~</h4>~~~
+@@long-footnote
+   Many existing uses of thread local state happen to be relatively robust and give correct answers only because the functions they are calling during execution do not yield. One may then think "well, I can just avoid this problem by making sure my code doesn't yeield", but we think this is a bad and unsustainable idea, because whether or not a function call will yield is not stable, obvious, or easily inspectable.
```

Comment:
I can't diff for some reason `yeield -> yield`

Reply:
oops, thanks